cdap-docs/admin-manual/source/security.rst

```diff
@@ -456,30 +449,27 @@ security components are working as expected:
 
     {"auth_uri":["http://localhost:10009/token"]}
 
-- Submit a username and password to one of the URIs to obtain an ``AccessToken``:
+- Submit a username and password to one of the authentication URIs (``<auth-uri>``) to
+  obtain an ``AccessToken``::
```

Comment:
So this is incorrect. You submit a Basic Authorization header. The API doesn't take username and password.